### PR TITLE
bun websockets

### PR DESCRIPTION
**Pull request by eabnelson**
example in stripe ui can be deleted

### CODE REVIEW

File: server/src/server.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+			return new Response('Not a WebSocket request.');
+		}
+	},
+	websocket: {
+		open(ws: ServerWebSocket) {
+			ws.subscribe('onlook-editor');
+			console.log('A new connection has been opened and subscribed to file updates.');
+		},
+		async message(ws: ServerWebSocket, message: string) {
+			await handleChanges(ws, message);
```
Comment (Kitenite) [inline server/src/server.ts:25]: What's the reason for the async?

File: server/src/server.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			// If upgrade succeeds, automatically returns a 101 Switching Protocols
+			return undefined;
+		} else {
+			// Handle HTTP request normally
+			return new Response('Not a WebSocket request.');
+		}
+	},
+	websocket: {
+		open(ws: ServerWebSocket) {
+			ws.subscribe('onlook-editor');
```
Comment (Kitenite) [inline server/src/server.ts:21]: So this is for 2-way message passing?

File: server/src/server.ts (hunk 1 above)
Comment (eabnelson) [reply server/src/server.ts]: ah thought I might have to await the writing to a file but guess not after some testing

File: server/src/server.ts (hunk 2 above)
Comment (eabnelson) [reply server/src/server.ts:21]: yep!

**Pull request merged**